dedicated-examples/extended-error-tracking/README.MD

```diff
@@ -0,0 +1,55 @@
+# Extended Client-Side Error Tracking
+
+When a JavaScript error occurs on a tested web page, TestCafe stops test execution and posts an error message to the console. Third-party code execution (e.g. controls, logging and analytics services) might cause unexpected errors that are not relevant to the tested app. You might want to filter those errors out to ignore them during the test.
+
+This example demonstrates how to expand on TestCafe's standard error logging and filter client-side errors that happen during a test.
```

Comment:
`This example demonstrates how to extend TestCafe's standard error tracking and filter...`

Reply:
done